app/src/main/java/com/framgia/music_44/screen/fragmentHome/FragmentHome.java

```diff
@@ -0,0 +1,48 @@
+package com.framgia.music_44.screen.fragmentHome;
+
+import android.content.ContentResolver;
+import android.content.ContentUris;
+import android.content.Context;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.framgia.music_44.R;
+import com.framgia.music_44.data.model.Songs;
+import com.framgia.music_44.screen.fragmentHome.adapter.AdapterHome;
+import com.framgia.music_44.screen.util.OnItemClickListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FragmentHome extends Fragment implements OnItemClickListener {
+    private View mView;
+    private List<Songs> mSongsList;
+    AdapterHome mAdapterHome;
+
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+            @Nullable Bundle savedInstanceState) {
+        mView = inflater.inflate(R.layout.fragment_home, container, false);
+        mSongsList = new ArrayList<>();
```

Comment:
tách method ra m.
nên phân vùng từng chức năng ra để tránh phình to method
```suggestion
        initView(view)
```